index.html

```diff
@@ -223,5 +223,49 @@ <h2>Skills</h2>
         </ul>
       </div>
     </section>
+
+    <footer id="contact-form" class="flex-container">
+      <h1>
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h1>
+
+      <form action="https://formspree.io/f/xeqdoera" method="POST">
+        <input type="text" name="fullname" id="fullname" placeholder="Full name" maxlength="30" required>
+        <input type="email" name="email" id="email" placeholder="Email address" required>
+        <textarea name="message" id="massage" maxlength="500" placeholder="Enter text here" required ></textarea>
+        <button class="call-to-action" type="submit">Get in touch</button>
+      </form>
+
+      <ul class="flex-container social-media">
+        <li>
+          <a href="https://twitter.com/hayet_fe">
+            <img src="images/twitter.png" alt="twiiter icon" class="icon"
```

Comment:
Great job @laiifuu 🤝

- [ ] As stated in the project requirements, kindly stick to the Figma design as much as possible.
- Your design
![Your design](https://user-images.githubusercontent.com/63917725/186192293-cdb04d9e-de0f-43e5-9db8-f3b540b2b089.PNG)

- Figma design
![Figma design](https://user-images.githubusercontent.com/63917725/186192387-543b00ff-633f-47e7-a735-1417eacec478.PNG)